series/type-string.go

```diff
@@ -43,6 +43,10 @@ func (e *stringElement) Set(value interface{}) {
 		}
 	case Element:
 		e.e = val.String()
+		if e.e == "NaN" {
```

Comment:
Can we use `Element.IsNA()` instead of detecting from its string representation?
Something like
```
e.nan = val.IsNA()
if e.nan != nil {
   e.e. = val.String()
}
```

Reply:
yes, that would be better